### ISSUE TITLE
Rendering MD-File from (E)Git-History causes ClassCastException

### ISSUE THREAD

**Issue by SchubP** [grosenberg/fluentmark#25]
Hi,
thanks for this cool plugin! :-)
But sadly we have a tiny problem.

We are using the following:
Eclipse SDK
Version: Oxygen.2 (4.7.2)
Build id: M20171130-0510
and these relevant Plugins:

- FluentMark Editor: 0.20.0.20180215-2100
- Eclipse EGit: 4.9.2.201712150930-r

When we open a readme file inside the worspace and view it in the FluentMark View everything works like expected.
Then we open the corresponding "History view" and open an old version of this readme file from our Git repo. This still works. But if we then open/show the FluentMark View, we will always get the error below:

```
!ENTRY org.eclipse.core.jobs 4 2 2018-02-16 09:57:50.888
!MESSAGE An internal error occurred during: "Update".
!STACK 0
java.lang.ClassCastException: org.eclipse.egit.ui.internal.revision.FileRevisionEditorInput cannot be cast to org.eclipse.ui.IPathEditorInput
	at net.certiv.fluentmark.editor.FluentMkEditor.getHtml(FluentMkEditor.java:573)
	at net.certiv.fluentmark.views.UpdateJob.run(UpdateJob.java:70)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
```

**Comment by grosenberg** [comment]
Version 0.20.0 does not have an `UpdateJob.java`.  Last version with that file was 0.15.0. So, something odd about your installation.

In any case, the error stems from an odd use of an internal class by Git - currently exploring an `Adaptor`-based fix.

**Comment by grosenberg** [comment]
Should be fixed in v0.22.0, just released.
